src/Chart.php

```diff
@@ -63,8 +72,12 @@ public function renderView(): void
     /**
      * @return array<string, mixed>
      */
-    public function getConfig(): array
+    protected function getConfig(): array
     {
+        if ($this->type === null) {
```

Comment:
remove exception and use `public string $type;` declaration (other untyped props are fine)